msa/black-box-tests/pom.xml

```diff
@@ -171,6 +171,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.thingsboard</groupId>
+            <artifactId>application</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why?

Reply:
Because without this dependency we have no access for necessary entities like EntityDataUpdate, UnreadNotificationsUpdate, AlarmCountUpdate etc. I think it will be better to have this dependency instead of having two similar entities in different places.